app/src/sharedTest/java/tech/relaycorp/gateway/test/CargoDeliveryCertPath.kt

```diff
@@ -0,0 +1,31 @@
+package tech.relaycorp.gateway.test
+
+import tech.relaycorp.relaynet.issueGatewayCertificate
+import java.time.ZonedDateTime
+
+/**
+ * Relaynet PKI certification path from a private gateway to a public gateway.
+ *
+ * See also [FullCertPath]
+ */
+object CargoDeliveryCertPath {
+    private val startDate = ZonedDateTime.now().minusMinutes(1)
+    private val endDate = ZonedDateTime.now().plusHours(1)
+
+    val PRIVATE_GW = issueGatewayCertificate(
```

Comment:
I'm afraid all this global `val`s will run whenever a test gets initiated. Can we use instead `val PRIVATE_GW by lazy { issueGatewayCertificate(...) }`? Trying to avoid making the tests slower

Reply:
Absolutely! I didn't realise it'd be re-evaluated.